tests/artifactorygroup_test.go

```diff
@@ -77,10 +74,8 @@ func testAddUsersToGroup(t *testing.T) {
 
 func testDeleteGroup(t *testing.T) {
 	groupParams := getTestGroupParams(false)
-	err := testGroupService.CreateGroup(groupParams)
-	assert.NoError(t, err)
-	err = testGroupService.DeleteGroup(groupParams.GroupDetails.Name)
-	assert.NoError(t, err)
+	assert.NoError(t, testGroupService.CreateGroup(groupParams))
+	assert.NoError(t, testGroupService.DeleteGroup(groupParams.GroupDetails.Name))
```

Comment:
Why not use the` deleteGroupAndAssert`?
Or delete the ` deleteGroupAndAssert` if you think it is not useful enough.